lib/API/JSON/ResponseBuilders/NewslettersResponseBuilder.php

```diff
@@ -198,7 +198,7 @@ private function buildQueue(SendingQueueEntity $queue) {
       'deleted_at' => ($deletedAt = $queue->getDeletedAt()) ? $deletedAt->format(self::DATE_FORMAT) : null,
       'meta' => $queue->getMeta(),
       'task_id' => (string)$task->getId(), // (string) for BC
-      'newsletter_id' => (string)$queue->getNewsletter()->getId(), // (string) for BC
+      'newsletter_id' => ($queue->getNewsletter() instanceof NewsletterEntity) ? (string)$queue->getNewsletter()->getId() : null, // (string) for BC
```

Comment:
How is it possible PHPStan doesn't scream? You shouldn't call the function twice - PHPStan doesn't know if there are any side-effects.

Correct way should be:
```
($newsletter = $queue->getNewsletter()) ? (string)$newsletter->getId() : null;
```